google-cloud-clients/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
This one should stay 2016

Reply:
You're right, sorry, I updated the wrong one. Should be fixed now.